unittest/tfreedv_mode_800XA.c

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------*\
+
+  FILE........: tfreedv_mode_800XA.c
+  AUTHOR......: Jeroen Vreeken
+  DATE CREATED: 24 May 2020
+
+  Tests specific for mode 800XA
+
```

Comment:
Perhaps call this tfreedv_800XA_rawdata.c of similar? It's not actually testing the 800XA (voice) mode, just the rawdata API of this mode.

Reply:
I was actually planning on expanding them further to also test the voice part (atleast to some degree) it was just the rawdata that I needed first.